sdk/tests/integration/BaseIntegrationTest.cc

```diff
@@ -0,0 +1,81 @@
+/*-
+ *
+ * Hedera C++ SDK
+ *
+ * Copyright (C) 2020 - 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#include "BaseIntegrationTest.h"
+#include "AccountId.h"
+#include "Client.h"
+#include "ECDSAsecp256k1PrivateKey.h"
+#include "ED25519PrivateKey.h"
+
+#include <filesystem>
+#include <fstream>
+#include <gtest/gtest.h>
+#include <nlohmann/json.hpp>
+#include <string>
+#include <string_view>
+#include <unordered_map>
+
+using json = nlohmann::json;
+using namespace std;
+
+namespace Hedera
+{
+//-----
+void BaseIntegrationTest::SetUp()
+{
+  const auto accountId = AccountId::fromString("0.0.3");
+  const std::string_view accountIdStr = "0.0.3";
```

Comment:
I don't think we want these hardcoded values moving forward. I think it'd be better for use to have processing that would loop through the "network" JSON value and build the network from there. If a configuration was fed into the integration tests that didn't use `0.0.3`, this would break.

Reply:
Agreed. This part has been reworked in 83259f5adc608fae6d8102294065ded2dacb5d3d.